index.html

```diff
@@ -0,0 +1,363 @@
+          </li>
+          <li class="about__icons__li">
+            <a href="https://twitter.com/irantsane" class="about__icons__link link"><img src="images/twitter.png" alt="twitter icon" /></a>
+          </li>
+          <li class="about__icons__li">
+            <a href="#" class="about__icons__link link"><img src="images/M.png" alt="M icon" /></a>
+          </li>
+        </ul>
+        <button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white">
+          Get my resume
+        </button>
+      </div>
+      <ul class="about__tools ul">
+        <li class="about__lang">
+          <div class="about__lang__top flex">
+            <h3 class="about__lang__top__title fw-500 fz-20 lh-24 clr-n-900">
+              Languages
+            </h3>
+            <img class="about__lang__top__img" src="images/arrow.png" alt="arrow icon" />
+          </div>
+          <ul class="about__lang__bottom ul">
+            <li class="flex bg-n-300">
+              <img src="images/js.png" alt="javascript icon" />
+              <p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">
+                JavaScript
+              </p>
+            </li>
+            <li class="flex bg-n-300">
+              <img src="images/html.png" alt="html icon" />
+              <p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">
+                HTML
+              </p>
+            </li>
+            <li class="flex bg-n-300">
+              <img src="images/css.png" alt="css icon" />
+              <p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">
+                CSS
+              </p>
+            </li>
+          </ul>
+        </li>
+        <li class="about__frameworks">
+          <div class="about__frameworks__top flex">
+            <h3 class="about__frameworks__top__title fw-500 fz-20 lh-24 clr-n-900">
+              Frameworks
+            </h3>
+            <img class="about__frameworks__top__img" src="images/arrow.png" alt="arrow icon" />
+          </div>
+        </li>
+        <li class="about__skills">
+          <div class="about__skills__top flex">
+            <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">
+              Skills
+            </h3>
+            <img class="about__skills__top__img" src="images/arrow.png" alt="arrow icon" />
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section class="contact" id="contact">
+      <div class="contact__container">
+        <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
+        <p class="contact__dsc fw-500 fz-20 lh-28">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I'd love
+          to help with it
+        </p>
+        <form id="form" action="#" method="post">
+          <ul class="ul">
+            <li>
+              <label for="fname" class="fw-500 fz-17 lh-20">Name:</label>
```

Comment:
- [ ] The form is looking good, however we should follow the design in Figma as much as possible. Kindly remove the labels to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200324292-2b27210d-c065-46ce-8b61-c79c02d0c991.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200324318-55b830cd-4011-4c5c-b7a7-acd91d07cddf.png)